Notation.Plugin.AzureKeyVault/Protocol/GenerateSignature.cs

```diff
@@ -9,22 +9,62 @@ namespace Notation.Plugin.Protocol
     public class GenerateSignatureRequest
```

Comment:
I think it works, but it may not help a lot because I still need to define the JsonPropertyName and manually do the the parameter validation during initialization.

Reply:
Thanks Shiwei! Updated to use `record` in KeyVaultClient.